### ISSUE TITLE
Configuration API

### ISSUE THREAD

**Issue by eldadfux** [appwrite/appwrite#323]
Adding a configuration API to allow client and backend apps to manage remote config with A/B test capabilities.

Should use a new controller under /v1/config

**Comment by christyjacob4** [comment]
Yes @eldadfux  it would be nice to keep this feature isolated from the rest of the endpoints

**Comment by christyjacob4** [comment]
What are the endpoints we're planning for this controller?

**Comment by eldadfux** [comment]
Not really sure yet, but I guess something like this:

POST /v1/config/[name-to-represent-a-set-of-settings]
GET /v1/config/[name-to-represent-a-set-of-settings]
GET /v1/config/[name-to-represent-a-set-of-settings]/:id
PUT /v1/config/[name-to-represent-a-set-of-settings]/:id
DELETE /v1/config/[name-to-represent-a-set-of-settings]/:id

**Comment by ItzNotABug** [comment]
Until this is added natively, here's a quick solution I built for mobile devs - https://github.com/itzNotABug/arc.

Though I'd really love to see this added to appwrite anytime now.
